frontend/src/pages/QADashboard/RecipientsWithNoTta/index.js

```diff
@@ -1,24 +1,54 @@
-/* eslint-disable no-alert */
-/* eslint-disable no-console */
 import React, {
-  useState,
+  useContext,
+  // useState,
```

Comment:
Some commented out lines not sure if intentional.

Reply:
I'll delete